client/components/global-notices-v2/notice.tsx

```diff
@@ -0,0 +1,44 @@
+import { Snackbar } from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { store as noticesStore } from '@wordpress/notices';
+import { useCallback, useEffect, useRef } from 'react';
+
+const Notice = ( { content, ...args }: { content: React.ReactNode; [ key: string ]: any } ) => {
+	const dismissTimeoutRef = useRef< null | ReturnType< typeof setTimeout > >( null );
+
+	const { removeNotice } = useDispatch( noticesStore );
+
+	const timeoutDuration = 6000;
```

Comment:
Some notices, historically, we kept visible for longer, some - for shorter. And some - until dismissed. Are we planning to make this one configurable?

Reply:
Yes, this is not currently possible with the way Core notices work. But, I think we will update the Core whenever we can rather than taking this route.